src/Razor/src/Microsoft.AspNetCore.Razor.Language/Legacy/ParserHelpers.cs

```diff
@@ -121,5 +121,3752 @@ public static bool IsDecimalDigit(char value)
+            { "&lbarr;", "\u290C" },
+            { "&#10508;", "\u290C" },
+            { "&lBarr;", "\u290E" },
+            { "&#10510;", "\u290E" },
+            { "&lbbrk;", "\u2772" },
+            { "&#10098;", "\u2772" },
+            { "&lbrace;", "\u007B" },
+            { "&#123;", "\u007B" },
+            { "&lbrack;", "\u005B" },
+            { "&#91;", "\u005B" },
+            { "&lbrke;", "\u298B" },
+            { "&#10635;", "\u298B" },
+            { "&lbrksld;", "\u298F" },
+            { "&#10639;", "\u298F" },
+            { "&lbrkslu;", "\u298D" },
+            { "&#10637;", "\u298D" },
+            { "&Lcaron;", "\u013D" },
+            { "&#317;", "\u013D" },
+            { "&lcaron;", "\u013E" },
+            { "&#318;", "\u013E" },
+            { "&Lcedil;", "\u013B" },
+            { "&#315;", "\u013B" },
+            { "&lcedil;", "\u013C" },
+            { "&#316;", "\u013C" },
+            { "&lceil;", "\u2308" },
+            { "&#8968;", "\u2308" },
+            { "&lcub;", "\u007B" },
+            { "&Lcy;", "\u041B" },
+            { "&#1051;", "\u041B" },
+            { "&lcy;", "\u043B" },
+            { "&#1083;", "\u043B" },
+            { "&ldca;", "\u2936" },
+            { "&#10550;", "\u2936" },
+            { "&ldquo;", "\u201C" },
+            { "&#8220;", "\u201C" },
+            { "&ldquor;", "\u201E" },
+            { "&ldrdhar;", "\u2967" },
+            { "&#10599;", "\u2967" },
+            { "&ldrushar;", "\u294B" },
+            { "&#10571;", "\u294B" },
+            { "&ldsh;", "\u21B2" },
+            { "&#8626;", "\u21B2" },
+            { "&le;", "\u2264" },
+            { "&#8804;", "\u2264" },
+            { "&lE;", "\u2266" },
+            { "&#8806;", "\u2266" },
+            { "&LeftAngleBracket;", "\u27E8" },
+            { "&LeftArrowBar;", "\u21E4" },
+            { "&leftarrow;", "\u2190" },
+            { "&LeftArrow;", "\u2190" },
+            { "&Leftarrow;", "\u21D0" },
+            { "&LeftArrowRightArrow;", "\u21C6" },
+            { "&#8646;", "\u21C6" },
+            { "&leftarrowtail;", "\u21A2" },
+            { "&LeftCeiling;", "\u2308" },
+            { "&LeftDoubleBracket;", "\u27E6" },
+            { "&#10214;", "\u27E6" },
+            { "&LeftDownTeeVector;", "\u2961" },
+            { "&#10593;", "\u2961" },
+            { "&LeftDownVectorBar;", "\u2959" },
+            { "&#10585;", "\u2959" },
+            { "&LeftDownVector;", "\u21C3" },
+            { "&LeftFloor;", "\u230A" },
+            { "&#8970;", "\u230A" },
+            { "&leftharpoondown;", "\u21BD" },
+            { "&leftharpoonup;", "\u21BC" },
+            { "&#8636;", "\u21BC" },
+            { "&leftleftarrows;", "\u21C7" },
+            { "&#8647;", "\u21C7" },
+            { "&leftrightarrow;", "\u2194" },
+            { "&LeftRightArrow;", "\u2194" },
+            { "&Leftrightarrow;", "\u21D4" },
+            { "&leftrightarrows;", "\u21C6" },
+            { "&leftrightharpoons;", "\
```

Comment:
Does this contain all decimal and hexidecimal entities? That seems like a lot to put in a static table when it's a mechanical translation to code points... https://en.wikipedia.org/wiki/Character_encodings_in_HTML

Reply:
I guess I only anticipated we'd need a lookup table for the XML entities since those don't have a regular mapping to code points.

Reply:
> Does this contain all decimal and hexidecimal entities

Good catch. It only contains decimals right now. I need to update it.

> I guess I only anticipated we'd need a lookup table for the XML entities since those don't have a regular mapping to code points.

 We still want to support all the named character entities that HTML5 supports right? Also what if the user types some unsupported hexadecimal and decimal entity? We probably don't want to translate in to a codepoint